files/__addonLocation__/package.json

```diff
@@ -29,12 +30,35 @@
   },
   "devDependencies": {
     "@babel/core": "^7.17.0",
-    "@babel/eslint-parser": "^7.18.2",
+    <% if (typescript) { %>"@babel/preset-typescript": "7.18.6"<% } else { %>"@babel/eslint-parser": "^7.18.2"<% } %>,
```

Comment:
Is there a reason we want to lock to this exact version of `@babel/preset-typescript`, or can there be a `^` here?

Reply:
Oh, no, that must have been accidental. Great catch! Fixed.